wiki/Installation-guide.md

```diff
@@ -22,8 +22,8 @@ For any problem visit the [Troubleshooting page](https://github.com/alvr-org/ALV
 
 ### Windows microphone streaming
 
-To use the microphone you need to install the [VB-CABLE driver](https://vb-audio.com/Cable/).
-Set "CABLE Output" as the default microphone.
+To use the microphone you need to install the [VB-CABLE driver](https://vb-audio.com/Cable).
+Set "VB Cable" as the default microphone.
```

Comment:
This needs to get changed back to `CABLE Output`, unless you can show us that VB-CABLE changed the naming of their devices. Also sorry that it took so long, we were waiting for the change on your end, but perhaps you missed it because it was in a different conversation.